src/main/kotlin/no/nav/familie/ef/sak/service/steg/BeslutteVedtakSteg.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.familie.ef.sak.service.steg
+
+import no.nav.familie.ef.sak.repository.domain.Behandling
+import no.nav.familie.ef.sak.service.FagsakService
+import no.nav.familie.ef.sak.sikkerhet.SikkerhetContext
+import no.nav.familie.ef.sak.task.IverksettMotOppdragTask
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.springframework.stereotype.Service
+
+@Service
+class BeslutteVedtakSteg(val taskRepository: TaskRepository, val fagsakService: FagsakService) : BehandlingSteg<Void?> {
+
+    override fun validerSteg(behandling: Behandling) {
+
+    }
+
+    override fun stegType(): StegType {
+        return StegType.BESLUTTE_VEDTAK
+    }
+
+    override fun utførSteg(behandling: Behandling, data: Void?) {
+        val fagsak = fagsakService.hentFagsak(behandling.fagsakId)
+        // Hvis godkjent
```

Comment:
// Hvis godkjent ? 
Hva menes med denne?

Reply:
Hvis beslutter godkjenner så skal dette skje.
Hvis beslutter underkjenner så må vi gjøre litt annet snacks :)

Reply:
(Dette er på ingen måte komplett implementasjon for godkjenning, bare at flyten tilsier gå til iverksettMotOppdrag hvis det blir godkjent... Må nok litt mer på plass her f.eks oppgave som skal ferdigstilles osv)